security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+.. _rbac_demo:
+
+Overview
+========
+
+This demo showcases the |rbac-long| functionality in |cp|.
+It is mostly for reference to see a workflow using the new |rbac| feature across the services in |cp|.
+
+There are two ways to run the demo.
+
+-  :ref:`Local install <rbac_demo_local>`
+-  :ref:`Docker <rbac_demo_docker>`
+
+
+Additional Reading
+------------------
+
+-  `RBAC for Kafka Connect whitepaper <https://www.confluent.io/resources/rbac-for-kafka-connect>`__
+-  `RBAC documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+
+
+.. _rbac_demo_local:
+
+Local install
+=============
+
+This method of running the demo is for users who have `downloaded <https://www.confluent.io/download/>`__ |cp| to their local hosts.
+
+Caveats
+-------
+
+-  For simplicity, this demo does not use LDAP, instead it uses the Hash
+   Login service with statically defined users/passwords. Additional
+   configurations would be required if you wanted to augment the demo to
+   connect to your LDAP server.
+-  The |rbac| configurations and role bindings in this demo are not
+   comprehensive, they are only for development to get minimum |rbac|
+   functionality set up across all the services in |cp|.
+   Please refer to the `RBAC
+   documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+   for comprehensive configuration and production guidance.
+
+Prerequisites
+-------------
+
+1. :ref:`Confluent CLI <cli-install>`:
+   |confluent-cli| must be installed on your machine, version
+   ``v0.127.0`` or higher (note: as of CP 5.3, the |confluent-cli| is a separate
+   :ref:`download <cli-install>`
+2. `Confluent Platform 5.3 <https://www.confluent.io/download/>`__: use
+   a clean install of |cp| without modified properties
+   files, or else the demo is not guaranteed to work
+3. ``jq``
+
+Run the demo
+------------
+
+#. Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__ repository from GitHub and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+       git clone git@github.com:confluentinc/examples.git
+       cd examples
+       git checkout |release_post_branch|
+
+#. Navigate to ``security/rbac/scripts`` directory.
+
+   .. codewithvars:: bash
+
+       cd security/rbac/scripts
+
+#. You have two options to run the demo.
+
+-  Option 1: run the demo end-to-end for all services
+
+   .. code:: bash
+
+      $ ./run.sh
+
+-  Option 2: step through it one service at a time
+
+   .. code:: bash
+
+      $ ./init.sh
+      $ ./enable-rbac-broker.sh
+      $ ./enable-rbac-schema-registry.sh
+      $ ./enable-rbac-connect.sh
+      $ ./enable-rbac-rest-proxy.sh
+      $ ./enable-rbac-ksql-server.sh
+      $ ./enable-rbac-control-center.sh
+
+#. After you run the demo, view the configuration files:
+
+   .. code:: bash
+
```

Comment:
suggest breaking up these 3 code examples so that can be copy/pasted into the terminal; and remove the `$`

Reply:
Ack